### PR TITLE
Add Base Preview Auto Select documentation

### PR DESCRIPTION
**Pull request by dacharyc**
Documentation for the Base Preview Auto Select functionality.

### CODE REVIEW

File: content/building-a-preview/work-with-base-previews/building-new-previews.md [hunk 1]
```diff
@@ -65,3 +65,57 @@ Select the {{% ui-text %}}Build with no base preview{{% /ui-text %}} option.
 ![Select Build with no base preview](/_images/select-build-with-no-base-preview.png)
 
 {{% /expand%}}
+
+## Build a Preview from a specific Base Preview
+
+If you want to use a Preview as a Base Preview for a specific Preview build, but don't want to set it as a [Repository]
```
Comment (bchavet) [inline content/building-a-preview/work-with-base-previews/building-new-previews.md:71]: [Repository] and [Branch Base Preview] are rendering as is here. Should they be links?

Screen Shot 2020-04-09 at 1 00 25 PM

File: content/troubleshooting/preview-built-problem.md [hunk 2]
```diff
@@ -69,6 +70,82 @@ runs config file commands from both `update` and `build`. You need to manually
 [Rebuild a Base Preview](/building-a-preview/work-with-base-previews/change-or-update/#change-a-base-preview) to pull a
 new Docker image. {{% /notice %}}
 
+## Using Base Previews - unexpected Preview build results
+
+When you're using Base Previews, you may experience a few different types of unexpected build results:
+
+- [A Preview that I didn't expect to see got built](#a-preview-that-i-didn-t-expect-to-see-got-built)
+- [A Preview that I did expect to see didn't build](#a-preview-that-i-did-expect-to-see-didn-t-build)
```
Comment (q0rban) [inline content/troubleshooting/preview-built-problem.md:78]: Thought:

- A Preview was built that I didn't expect
- A Preview I expected to build is missing

File: content/building-a-preview/work-with-base-previews/building-new-previews.md (hunk 1 above)
Comment (dacharyc) [reply content/building-a-preview/work-with-base-previews/building-new-previews.md]: Aww, it was kind of you to attribute this to a rendering issue. I just missed adding the links ;) Told you my brain was melting. Well spotted, thanks, @bchavet !

File: content/troubleshooting/preview-built-problem.md (hunk 2 above)
Comment (dacharyc) [reply content/troubleshooting/preview-built-problem.md]: Thanks for the suggestions, @q0rban ! I did think the wording here was awkward, but it was the last thing I was trying to finish up and my brain couldn't come up with anything better. This is an improvement, so I've borrowed and tweaked a little. Thank you!

**Pull request merged**